### PR TITLE
Update ISSUE_TEMPLATE.md

### PR DESCRIPTION
**Pull request by asettle**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -35,3 +35,12 @@ the backtick lines:
 ## What did you expect to happen instead of what actually happened?
 
 > Example: I expected a cinder volume snapshot to be successfully created.
+
+## Do you think the documentation team wants to know about this?
+[ ] Yes, and I submitted an issue to [Docs issues](https://github.com/rackerlabs/docs-rpc/issues "Docs issues")
```
Comment (major) [inline .github/ISSUE_TEMPLATE.md]: Put a blank line before your check boxes so they render properly. :)

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -35,3 +35,12 @@ the backtick lines:
...
+[ ] Yes, and I contributed a documentation patch to [docs-rpc](https://github.com/rackerlabs/docs-rpc "docs-rpc")
+[ ] No documentation required for this change
+
+## Does your change require a release note?
+[ ] Yes, it's a pretty big deal like I am and I've made a reno note!
```
Comment (major) [inline .github/ISSUE_TEMPLATE.md]: Blank line here, too.

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -35,3 +35,14 @@ the backtick lines:
 ## What did you expect to happen instead of what actually happened?
 
 > Example: I expected a cinder volume snapshot to be successfully created.
+
+## Do you think the documentation team wants to know about this?
+
+[ ] Yes, and I submitted an issue to [Docs issues](https://github.com/rackerlabs/docs-rpc/issues "Docs issues")
```
Comment (major) [inline .github/ISSUE_TEMPLATE.md]: Aaack, I forgot that the checkboxes need a dash up front >> `- [ ] I LIKE PANCAKES`

**Comment by antonym** [comment]
recheck_all

**Comment by asettle** [comment]
recheck_all

**Comment by asettle** [comment]
recheck_all

**Comment by mattt416** [comment]
recheck_cit_swift

**Pull request merged**